_opensearch/query-dsl/text-analyzers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Text analyzers
+parent: Query DSL
+nav_order: 41
+---
+
+
+# Text analyzers
+
+OpenSearch applies text analysis during indexing or searching for `text` fields. There is a standard  analyzer that OpenSearch uses by default for text analysis. To optimize unstructured text for search, you can convert it into structured text with our specialized text analyzers.
+## Specialized analyzers
+
+OpenSearch provides specialized analyzers to convert your structured text into the format that works best for your searches.
+
+OpenSearch supports the following specialized text analyzers:
+
+1. **Standard analyzer** – parses text strings into terms at word boundaries per the Unicode text segmentation algorithm. It also removes stop words and removes punctuation and lowercase terms.
+1. **Simple analyzer** – converts all characters to lower case and divides groups of non-letter characters into terms.
+1. **Whitespace analyzer** – parses characters into terms between whitespaces.
+1. **Stop analyzer** – Removes stop words (e.g. "but," or "this") from the query string.
+1. **Keyword analyzer** – receives text and outputs only the text specified as a single keyword term.
+1. **Pattern analyzer** – splits text into terms using a regular expression and supports lower case characters and stop words.
```

Comment:
... supports lowercasing the string and removing stop words

Reply:
we can't say "lowercasing" that's not a verb.